UI/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>
+        Store Manager
+    </title>
+    <link rel="stylesheet" href='static/css/style.css'>
+    <script src="static/CSS/main.js"></script>
+</head>
+
+<body>
+    <div class="container">
+        {% block navigation %}
+        <header>
+            <div class="header">
+                    <div class="container" onclick="myFunction(this); todo();">
+                            <div class="bar1"></div>
+                            <div class="bar2"></div>
+                            <div class="bar3"></div>
+                        </div>
+            <div class="link">
+               <a href="#" name="store-manager">Store-Manager</a>
+            </div>
+                <form class="search">
+                    <input class="searchinput" type="text" placeholder="Search for products">
+                    <button class="btn">search</button>
+                
+                <div class="header_buttons">
+                    <a href = "cart.html" class="btn bg-slate login_button"> Cart
+                    <img src="static/images/cart.png" alt="cart" class="cart"></a>
+                   <a href="profile.html" class="btn bg-slate login_button">Profile
+                        <img src="static/images/profile.png" alt="profile" class="cart">
+                   </a>
+                   <a href="login.html" class="btn bg-slate login_button">LOGOUT
+                        <img src="static/images/logout.jpg" alt="logout" class="cart"></a>
+                </div>
+            </form>
+            </div>
+        </header>
+        {% endblock %}
+        
+        {% block sidebar %} 
+        <div id="mySidenav" class="sidenav">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <a href="#">Home</a>
+              <div class="dropdown">
+                <a href="/UI/products.html" class="dropbtn">Products</a>
+                <div class="dropdown-content">
+                        <a href="#">Categories</a>
+                        <a href="#">Toiletries</a>
+                        <a href="#">Foodstuffs</a>
+                        <a href="#">Drinks</a>
+                        <a href="#">Electronics</a>
+                </div>
+            </div>
+                <a href="#">Sales records</a>
+                <a href="#">Inventory</a>
+                <a href="#">Employee</a>
+        </div>
+        {% endblock %}
+          <!-- Page content -->
+          <div class="main">
+                {% block content %}{% endblock %}
+          </div>
+
+        <div class="footer">
+            {% block footer %}
```

Comment:
Are you using flask

Reply:
Nope
Just tried using base template didn't work though